lib/vehicle_factory.rb

```diff
@@ -0,0 +1,20 @@
+class VehicleFactory
+  def initialize
+    
+  end
```

Comment:
If you aren't doing anything in your `initialize` method, you can actually leave it out. Ruby will still be able to instantiate without it.